de.dlr.sc.virsat.model.calculation/src/de/dlr/sc/virsat/model/calculation/compute/IncrementalEquationBuilder.java

```diff
@@ -348,8 +349,37 @@ public boolean visit(IResourceDelta delta) throws CoreException {
 		}
 	
 		return equations;
+	}
+
+	/**
+	 * Writes all errors from resourceSet to log
+	 * @param resourceSet resource set to get errors from
+	 */
+	private void reportResourceSetErrors(VirSatResourceSet resourceSet) {
+		for (Resource resource : resourceSet.getResources()) {
+			for (Resource.Diagnostic error : resource.getErrors()) {
+				Status status = getResourceErrorStatus(resource, error);
+				Activator.getDefault().getLog().log(status);
+			}
+		}
 	};
 	
+	/**
+	 * Creates a status for an error in a resource
+	 * @param resource EMF resource with an error
+	 * @param error 
+	 * @return status for writing to log
+	 */
+	private Status getResourceErrorStatus(Resource resource, Resource.Diagnostic error) {
+		Status status;
+		Throwable exception = error instanceof Throwable ? (Throwable) error : null;
+		status = new Status(Status.ERROR, Activator.getPluginId(),
+				String.format("IncrementalEquationBuilder: error in resource %s: %s", resource.toString(),
+						error.getMessage()),
```

Comment:
May create a NPE in case that error is not of type throwable.

Reply:
If error is not throwable, exception will be null, which is accepted in `Status` constructor:

> **exception** a low-level exception, or `null` if not applicable

(from Status javadoc)